tests/behaviour/steps/inspectcli.py

```diff
@@ -484,8 +493,8 @@ def step_impl(context):
         "http://purl.org/linked-data/sdmx/2009/attribute#unitMeasure",
         ComponentPropertyType.Attribute,
         "",
-        "",
-        "Some Obs Val, Some Other Obs Val",
+        "Some Unit",
+        "Some Other Obs Val, Some Obs Val",
```

Comment:
This test seems like it might be a little flakey if the other in which things are concatenated keepschanging. I wonder if it's possible to sort the items that go into an aggregate like this.